File: src/Test2/Probability.py

def main():
    li = input().split(" ")
    n= int(li[0])
    x= int(li[1])
    prob = probability(n,x)
    print(prob)

def probability(n,x):
    Ns = factor(8)/(factor(n)*factor(8-n))
    Ne1= factor(4)/(factor(x)*factor(4-x))
    Ne2= factor(4)/(factor(n-x))*factor(4-(n-x))
    pe=int(((Ne1*Ne2)/Ns)*100)
    return pe

def factor(f):
    fact =1
    for i in range(2,f+1):
        fact*=i
    return fact


main()

File: src/Test2/Probability.java
//public class Solution {
//
//    public static int probability(int n,int x){
//
//        double Ns,Ne1,Ne2;
//        int Pe;
//        Ns=factorial(8.0)/(factorial(n*1.0)*factorial(8.0-n));
//        Ne1=factorial(4.0)/(factorial(x*1.0)*factorial(4.0-x));
//        Ne2=factorial(4.0)/(factorial(((n-x)*1.0))*factorial((4.0-(n-x))));
//        Pe=(int)(((Ne1*Ne2)/Ns)*100);
//        return Pe;
//
//        /* Your class should be named Solution
//         * Don't write main().
//         * Don't read input, it is passed as function argument.
//         * Return output and don't print it.
//         * Taking input and printing output is handled automatically.
//         */
//
//    }
//
//    public static double factorial(double num){
//        int fact=1;
//        for(int i=2;i<=num;i++){
//            fact*=i;
//        }
//        return fact;
//    }
//
//
//}
